integrations/extensions/starter-kits/language-model-complex-informational-tasks/README.md

```diff
@@ -0,0 +1,68 @@
+| Invoke GPT Chat Completion API with Messages | Connects to OpenAI with the content configured by the "Generate Answer" action and asks the language model to generate an answer to the user query. It is not meant to be invoked directly, but rather by the "Generate Answer" action. This action is slightly different from the "Invoke GPT Chat Completion API" used in the [generic OpenAI starter kit](../language-model-openai/README.md) because it invokes the model with prompts and content bundled into `messages`, rather than with this expression `{"role": "user", "content": "${model_prompt}"`}`. |
+
+#### Session variables
+
+Below is a list of the session variables used in this example. Most of them are automatically set with defaults in the sample [discovery-openai-actions.json](discovery-openai-actions.json), so you do not need to set them yourself unless you want to make changes. You must, however, [set](https://cloud.ibm.com/docs/watson-assistant?topic=watson-assistant-manage-info#store-session-variable) the `discovery_project_id` to point to the project id for your Watson Discovery collection, as noted above.
+
+- `internet_plan_messages`: Holds information from the Internet Plan Questions action that is used as input to the model.
+- `internet_plan_prompt`: The prompt for the OpenAI model on the topic of internet plans. Specifically, this prompt is "You are an agent for a cable provider that helps customers determine which internet plan is best for them. Limit your response to 30 words and only discuss which plans are best for that customer unless prompted otherwise. The three plans are 1) The 100 Mbps plan which cost $30; 2) The 500 Mbps which cost $40; 3) The 1 Gbps plan which cost $50"
+- `last_action_accessed`: Used by "No Action Matches" to support multiple conversational turns on the same topic. For example the "Internet Plan Questions" action sets this value to "Internet Plan Questions" so after the first internet plan query, all subsequent queries will be fall back to the Internet Plan Questions action if they do not directly match some other action. This is meant to illustrate extended conversational interaction. It is a current limitation of the starter kit that once this value is set, it does not automatically get reset when the conversation moves on past this topic.
+- `messages` : Input to the OpenAI model. This includes the instructions and data supplied the plan prompt and the user query. In the sample actions file, messages look like this `[{"role": "user", "content": “${plan_messages}”}]`
```

Comment:
I think this last sentence is wrong.  The messages has multiple entries by the time it is sent to the OpenAI model.  You could might want to just drop the sentence.